src/polyrith.lean

```diff
@@ -224,7 +244,9 @@ by ring_nf; simp only [complex.I_sq]; ring
 
 -- # main tactic
 
-meta def polyrith : tactic unit :=
+declare_trace polyrith
```

Comment:
I found it useful to be able to trace the code sent to Sage. This lets you write `set_option trace.polyrith true` to see that. The behavior isn't right for production. (Right now, setting the trace option will *only* print the Sage command, and won't run the rest of the tactic.) But it helps for debugging.